server.go

```diff
@@ -99,7 +100,9 @@ func (users *Users) UpdateUserInfo(anUser *User) bool {
 	defer users.UserListMutex.Unlock()
 	users.UserList[anUser.Email].Update(anUser)
 	return true
+	// Comment
```

Comment:
Would be any benefit of returning the `user` itself instead of `bool` in here?

Reply:
Yes, it's actually quite common to return the value given in argument so the function could be chained. In this case I decided just return true if the user info were applied. Which is later interpreted as "user modified or not".